Earthfile

```diff
@@ -53,3 +51,30 @@ build:
     ENV CGO_ENABLED=0
     RUN go build -o enki -ldflags "${LDFLAGS}" main.go
     SAVE ARTIFACT enki enki AS LOCAL build/enki
+
+build-iso:
+    FROM +enki-image
+    ARG BASE_IMAGE=quay.io/kairos/ubuntu:24.04-core-amd64-generic-v3.1.1-uki
```

Comment:
should this one be defined at the top with the comment to be automagically upgraded?

Reply:
Good one!

Reply:
umm regex still doesnt look like its getting the other value in the Earthfile so this may need a rework later

Reply:
ok, I think I fixed it